prow/cmd/tackle/main.go

```diff
@@ -161,6 +175,55 @@ func selectProject(choice string) (string, error) {
 	return choice, nil
 }
 
+//selectZone returns the user-selected zone, defaulting to the current gcloud one.
+func selectZone() (string, error) {
+	const MaxZones = 20
+	var choice string;
+
+	def, err := currentZone()
```

Comment:
If they have a zone default, are you sure we want to ask them to confirm the zone vs just using it?

Reply:
The rationale here was that since the default zone is *global*, there may be a case where a user *may* want to override this at the cluster-level and not necessarily blindly use this global *default*. I erred on the side of caution here by giving the choice, but if it makes more sense to skip the prompt in cases where if is set that would make sense as well.

Reply:
Personally I like the explicit prompt with a more sane default, vs. just using the default without asking

Reply:
SGTM, `/hold cancel` when/if you are ready for this to merge!